provisioner/clusterpy.go

```diff
@@ -334,15 +367,18 @@ func (p *clusterpyProvisioner) Decommission(cluster *api.Cluster, channelConfig
 		logger.Error("Unable to downscale the deployments, proceeding anyway: %s", err)
 	}
 
+	// we don't support cancelling decommission operations yet
```

Comment:
Why not? Doesn't this violate the principle of least surprise? It should just work as CLM can already die at any moment.